integration-tests/test/http_client.py

```diff
@@ -8,7 +8,7 @@
 
 class HttpRequestException(Exception):
    def __init__(self, status_code: int, content: str):
-       super(HttpRequestException, self).__init__()
+       super().__init__()
```

Comment:
Are these changes to int tests relate to alert fix?

Reply:
These are related to the updated package pylint. New check feature was introduced.